packages/react/src/Button/Button.module.css

```diff
@@ -25,8 +26,8 @@
 
 .Button__text {
   position: relative;
-  top: 0.5px; /* visual offset for off-center mona sans glyphs */
-  line-height: var(--brand-text-lineHeight-200);
+  /*top: 0.5px;  visual offset for off-center mona sans glyphs */
+  /* line-height: var(--brand-text-lineHeight-200); */
```

Comment:
Could we add a comment explaining why these are commented out? Or remove these lines if no longer needed?